gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -156,8 +167,22 @@ def step(self, action, verbose=True):
                     inventory_changed_str = ''
                 print('{}: {}. {}'.format(
                     action_str, interaction_obj['objectType'], inventory_changed_str))
+        elif 'Rotate' in action_str:
```

Comment:
I would also take all of this rotation logic into a function outside from step and call it here

Reply:
its around 10 lines, other actions are around 10 lines as well, it doesn't deserve it's own function or if it does, we need to split step into around 6 functions

Reply:
Ok, we agreed to do this in another PR

Reply:
Left for another PR